docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -33,6 +33,12 @@ $ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
 
 For more information, see the GitHub documentation of the link:https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions[docker machine KVM driver].
 
+KVM has a standing issue. You might face an error like
+----
+error: Network not found: no network with matching name 'default'
+----
+If so, start a default network using virt-manager(virt-manager > edit > connection details > virtual networks > default), more link:https://github.com/kubernetes/minikube/issues/828[here]
+
```

Comment:
This should go the [troubleshooting guide](https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-driver-plugins.html) i.e. https://github.com/minishift/minishift/blob/master/docs/source/troubleshooting/troubleshooting-driver-plugins.adoc

Reply:
s/a/the
```diff
- virt-manager(virt-manager > edit > connection details > virtual networks > default)
+virt-manager (virt-manager > edit > connection details > virtual networks > default)
```
s/more here/For more informatioon, see link:...[...]

Would it be possible to refer to a real resource that explains how to active the default network, or how to restore it?

I would also prefer to see instructions from the command line when possible, such as:
```
virsh net-undefine default
virsh net-destroy default
virsh net-list
virsh net-define /usr/share/libvirt/networks/default.xml
virsh net-start default
virsh net-autostart default
```